### DISCUSSION TITLE
Estructuras parte 1

### DISCUSSION THREAD

**Discussion by antoniablanco** [IIC2133-PUC/2024-1#135] [Tarea 2] [answered]
Hola, ¿quería preguntar si es posible utilizar estructuras de string diferentes a hashing para la pregunta 1?.

**Comment by benjavicente** [comment, marked as answer]
Hola! Se recomienda usar hashing dado que proveen una forma eficiente de resolver el problema empleando contenidos del curso. Si quieres usar otra técnica, deberá tener igual o mejor complejidad a la que se espera con hashing.

**Comment by antoniablanco** [comment]
Súper, gracias!

Obtener Outlook para iOS<https://aka.ms/o0ukef>
________________________________
De: Benjamín Vicente ***@***.***>
Enviado: Friday, May 17, 2024 4:35:27 PM
Para: IIC2133-PUC/2024-1 ***@***.***>
Cc: Antonia Francisca Blanco Del Río ***@***.***>; Author ***@***.***>
Asunto: Re: [IIC2133-PUC/2024-1] Estructuras parte 1 (Discussion #135)


Hola! Se recomienda usar hashing dado que proveen una forma eficiente de resolver el problema empleando contenidos del curso. Si quieres usar otra técnica, deberá tener igual o mejor complejidad a la que se espera con hashing.
